config.yaml.in

```diff
@@ -27,17 +27,20 @@ languages:
     weight: 1
     contentDir: content/en
     < content/en/config.yaml >
+    < content/en/tabcontents.yaml >
   
   # Portuguese
   pt:
     title: NumPy
     weight: 2
     contentDir: content/pt
     < content/pt/config.yaml >
+    < content/en/tabcontents.yaml >
```

Comment:
The fix happened after adding this line here. Should I use automatic translation to generate content for `pt/tabcontent.yaml` and `ja/tabscontent.yaml` or should I just leave the text empty with the keys added. As of now these files are absent.

Reply:
Ah that makes sense. I'd say just copy over the English `.yaml` file to `ja/` and `pt/`, that will then make the files available for translation after merging this PR.